app/controllers/pages_controller.rb

```diff
@@ -4,7 +4,7 @@ def index
     @metadata.og.description = 'Interaktívne návody, ako vybaviť úradné záležitosti elektronicky.'
 
     @faqs = Page.faq.all # TODO: fetch top FAQs here
-    @journeys = Journey.published
+    @categories = Category.featured.order(name: :asc)
```

Comment:
Category nema nejaky position?

Reply:
nemá. Treba to tam dorobiť?

Reply:
@celuchmarek jasne dajme nech mame nad tym kontrolu.